test/check-application

```diff
@@ -73,9 +73,6 @@ class TestApplication(testlib.MachineCase):
 
         self.addCleanup(self.machine.execute, "killall -9 podman || true")
 
-        # HACK: https://github.com/containers/podman/issues/8762
-        self.admin_s.execute("timeout 5 curl --unix-socket $XDG_RUNTIME_DIR/podman/podman.sock http://d/v1.24/libpod/info || true")
```

Comment:
Curious, isn't that pretty much exactly the same workaround that the previous commit addresses? (info not working the first time)

Reply:
it indeed is.
There has been at least 3 times in last few months a problem, where the first call either is stuck or fails. Always slightly differently and the reproducer is always slightly different but takes me at least a full day to get to it. So this PR is me throwing hands in the air and just doing the big and ugly workaround with "if it does not work automatically as it should, just click the button to start it - as users would, if this would happen to them".